### ISSUE TITLE
CAMS2_82 new CNEMC reader - Chilean observations

### ISSUE THREAD

**Issue by andagit** [metno/pyaerocom#1082]
**Is your feature request related to a problem? Please describe.**
CAMS2_82 have additional observation data to be included in the IFS analysis

**Describe the solution you would like to see**
Data is downloaded and aggregated (as MEP) to: ~/aerocomB/aerocom1/AEROCOM_OBSDATA/CNEMC/aggregated/
Data is in HARP format - should be very similar to MEP. 
I have checked the aggregated NetCDF files and they are similar to MEP, but are missing some <cell_methods> with the variables, but not sure if this is crucial (or at what stage of the preprossesing of MEP data this is added? I tried to follow the same aggregate.sh script...)
Some of the variables CO,NO2,O3,SO2 are volume_mixing_ratio and not density - as PM10_density and PM2p5_density, but I think it is basically PM10 and PM2p5 values contained in the CNEMC files. (maybe some ozone perhaps).

**Additional context**
Need to implement some code to PyAerocom to read the files "sinca-*" starting from 202209.
I am willing to do something, but is a little bit rusty in the procedures on how to implement into Pyaerocom code.

**Comment by jgriesfeller** [comment]
Anna and I just had a quick look the the data. They are basically the same as MEP besides that fact that the species outside of PM10 and PM25 volume mixing ratios (unit ppm), while MEP provides densities (unit ug m3). There's no presseure / Temperature for conversion. But these are allways NaN anyway, so not important for now.
With the right renaming, these files could be read as MEP for PM10 and PM25, but we might need them as a separate obs network, right?
In that case, we need a new reader, which means we need  a HARP reader in pyaro and the parallelization needs to be tested with pyaro data sets (this has not been done so far).
So in short: All of that is doable, but will take some days of work. 
The problem is that Anna officially needs the results by 1st May...

**Comment by jgriesfeller** [comment]
solved via https://github.com/metno/pyaro-readers/issues/37

**Comment by jgriesfeller** [comment]
solved via https://github.com/metno/pyaro-readers/issues/37
